### PR TITLE
Improve docstrings for captions

### PR DESCRIPTION
**Pull request by sciunto**

### CODE REVIEW

File: doc/examples/color_exposure/plot_ihc_color_separation.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Immunohistochemical staining colors separation
 ==============================================
 
+Color deconvolution consists in the separation of features by their colors.
```
Comment (sciunto) [inline doc/examples/color_exposure/plot_ihc_color_separation.py]: oops typo

File: doc/examples/edges/plot_contours.py [hunk 2]
```diff
@@ -3,10 +3,10 @@
 Contour finding
 ===============
 
-``skimage.measure.find_contours`` uses a marching squares method to find
-constant valued contours in an image.  Array values are linearly interpolated
-to provide better precision of the output contours.  Contours which intersect
-the image edge are open; all others are closed.
+A marching squares method can be used to find constant valued contours in an
```
Comment (jni) [inline doc/examples/edges/plot_contours.py]: I don't like this change... Changes active to passive and no longer points to the function in skimage that does this.

**Comment by jni** [comment]
@sciunto great changes except one! =)

**Comment by sciunto** [comment]
@jni I made a fix. ;)

Happy sprinting :)

**Comment by jni** [comment]
👍 !

**Comment by sciunto** [comment]
@soupault Looks good for you as well?

**Comment by soupault** [comment]
@sciunto Yep, thanks!

**Comment by sciunto** [comment]
thanks @soupault and @jni :)

**Pull request merged**